build/build.zig

```diff
@@ -98,13 +110,23 @@ pub fn add_firmware(
     // On demand, generate chip definitions via regz:
     const chip_source = switch (chip.register_definition) {
         .json, .atdf, .svd => |file| blk: {
-            const regz_exe = mz.dependency("microzig/tools/regz", .{ .optimize = .ReleaseSafe }).artifact("regz");
+            const regz_exe = mz.dependency("microzig/tools/regz", .{ .optimize = .Debug }).artifact("regz");
```

Comment:
Intentional?

Reply:
Yes! Since regz isn't run very often, I think we'll end up wasting less time if we don't optimize regz, and run it in debug mode. On first run of the project the release build time is significant.